.github/workflows/ci.yml

```diff
@@ -116,16 +116,26 @@ jobs:
       - name: Test (${{ matrix.os }})
         run: cargo test
 
-      - name: Setup Android test environment
+  test_android:
+    name: "Test (Android)"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Enable KVM
+        run: |
+          echo 'KERNEL=="kvm", GROUP="kvm", MODE="0666", OPTIONS+="static_node=kvm"' | sudo tee /etc/udev/rules.d/99-kvm4all.rules
+          sudo udevadm control --reload-rules
+          sudo udevadm trigger --name-match=kvm
```

Comment:
This is sufficiently magical seeming it might be worth leaving a pointer comment to [the android-emulator-runner README](https://github.com/ReactiveCircus/android-emulator-runner?tab=readme-ov-file#running-hardware-accelerated-emulators-on-linux-runners) where it documents this setup. WDYT?

Reply:
I agree, good callout. I've updated this with two references to elsewhere about how to implement this.